idp-core/src/main/java/io/mosip/idp/core/dto/DiscoveryResponse.java

```diff
@@ -1,42 +0,0 @@
-/*
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at https://mozilla.org/MPL/2.0/.
- */
-package io.mosip.idp.core.dto;
-
-import lombok.Data;
-
-import java.util.List;
-
-@Data
-public class DiscoveryResponse {
```

Comment:
we have not removed the discovery API, I have only removed the DTO as we don't use DTO to create discovery response. Instead, i have created this property "mosip.idp.discovery.key-values"  in properties file. so that any change in the discovery response will not any code change.

mosip.idp.discovery.key-values={'issuer': '${mosip.idp.discovery.issuer-id}' ,\
  \ 'authorization_endpoint': '${mosipbox.public.url}${server.servlet.path}/authorize' , \
  \ 'token_endpoint': '${mosipbox.public.url}${server.servlet.path}/oauth/token' ,\
  \ 'userinfo_endpoint' : '${mosipbox.public.url}${server.servlet.path}/oidc/userinfo' , \
  \ 'introspection_endpoint' : '${mosipbox.public.url}${server.servlet.path}/oauth/introspect' , \
  \ 'jwks_uri' : '${mosipbox.public.url}${server.servlet.path}/oauth/jwks.json' , \
  \ 'scopes_supported' : ${mosip.idp.supported.openid.scopes}, \
  \ 'response_types_supported' : ${mosip.idp.supported.response.types}, \
  \ 'response_modes_supported' : { 'query' }, \
  \ 'token_endpoint_auth_methods_supported' : ${mosip.idp.supported.client.auth.methods}, \
  \ 'token_endpoint_auth_signing_alg_values_supported' : {'RS256'}, \
  \ 'userinfo_signing_alg_values_supported' : {'RS256'}, \
  \ 'userinfo_encryption_alg_values_supported' : {'RSAXXXXX'},\
  \ 'userinfo_encryption_enc_values_supported' : {'A128GCM'}, \
  \ 'id_token_signing_alg_values_supported' : {'RS256'}, \
  \ 'claim_types_supported': {'normal'}, \
  \ 'claims_parameter_supported' : true, \
  \ 'display_values_supported' : ${mosip.idp.supported.ui.displays}, \
  \ 'subject_types_supported' : { 'pairwise' }, \
  \ 'claims_supported' : {'iss','sub','acr','name','given_name','middle_name','preferred_username','picture','gender','birthdate','locale','nickname', 'family_name','zoneinfo', 'updated_at','address', 'email','email_verified', 'phone_number','phone_number_verified'}, \
  \ 'acr_values_supported' : {},\
  \ 'request_parameter_supported' : false, \
  \ 'ui_locales_supported' : {} }